app/services/promotional_tweet/post_promotional_tweet.rb

```diff
@@ -0,0 +1,42 @@
+class PostPromotionalTweet
+
+  def initialize(promotional_tweet, image, brand)
+    @message = promotional_tweet.message
+    @image   = image
+    @brand   = brand
+    @client  = brand.twitter_rest_client
+  end
+
+  def send!
+    @tweet = post_promo_tweet
+
+    if @tweet.present?
+      @promotional_tweet.update_attribute!(:tweet_url, @tweet.id)
```

Comment:
This column seems misnamed. Should we call it `tweet_id`? Also, we'll need to populate the page with a list of promotional tweets that were sent out. The other thing we should store so that we don't have to get it from the API is the datetime when the tweet was posted.

If we don't want to create multiple columns for all this data, we can create a json column for the relevant tweet data that we're going to need. I'd say we store the id, the url, and the datetime of when it was posted.